app/models/refinery/setting.rb

```diff
@@ -30,6 +30,10 @@ def per_page
         12
       end
 
+      def form_value_types
```

Comment:
you might want to update the form to use this version so that if it's overridden then it does something.

Reply:
Hahaha! You're right i've forgot to use it in the view.

Reply:
That's fixed now.